CHANGELOG.rst

```diff
@@ -98,7 +96,7 @@ Bug Fixes
 
 * Change capture.py's ``DontReadFromInput`` class to throw ``io.UnsupportedOperation`` errors rather
   than ValueErrors in the ``fileno`` method (`#2276`_).
-  Thanks `@metasyn`_ for the PR.
+  Thanks `@metasyn`_ and `@vlad-dragos`_ for the PR.
```

Comment:
If I'm reading #1967 and #2313 this was the same change, once to `master`, once to `features`, by two different people - correct? 😆